river/command/output.zig

```diff
@@ -93,6 +95,13 @@ fn getOutput(seat: *Seat, str: []const u8) !?*Output {
         ) orelse return null;
         return @intToPtr(*Output, wlr_output.data);
     } else {
+        // Check if an output matches by name
+        var it = server.root.outputs.first;
+        while (it) |node| : (it = node.next) {
+            if (std.mem.eql(u8, std.mem.sliceTo(node.data.wlr_output.name, 0), str)) {
```

Comment:
We actually don't need `sliceTo()` here as `wlr_output.name` is now a 0 terminated pointer not an array.
```suggestion
            if (std.mem.eql(u8, std.mem.span(node.data.wlr_output.name), str)) {
```

Reply:
And looks like I forgot to change the doc comment in zig-wlroots...

Reply:
Ah nice!